main_8_5.py

```diff
@@ -1712,8 +1712,8 @@ def javaScriptConsoleMessage(self, level, message, lineNumber, sourceID):
 
 
         self.panels_name = ["BAF3","BLANK","LAF3","MP1H","MP2D","MP2X","OS5","PAF3","SC","SM2","TP1H","TP2X","TPAX","WS4"]
-
-        self.SaveChanges_button.clicked.connect(self.SaveChanges_button_fun)
```

Comment:
Wtf ???
Why you changed this ?
I told you connect it to new button in second tab !!